backend/routes/goals.go

```diff
@@ -0,0 +1,298 @@
+package routes
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"database/sql"
+	"github.com/benzend/goalboard/database"
+	"github.com/golang-jwt/jwt/v5"
+)
+
+type setGoal struct {
+    Name            string `json:"Name"`
```

Comment:
`json:"name"`

if you leave it at `json:"Name"` the json obj will look like

```json
{ "Name": "foo" }
```

Which is weird json property naming